<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewCourse</class>
 <widget class="QDialog" name="NewCourse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Course</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="semesterLbl">
       <property name="text">
        <string>Semester:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="semesterBox">
       <item>
        <property name="text">
         <string>2020 Fall</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2020 Spring</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2019 Fall</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2019 Spring</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2018 Fall</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2018 Spring</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2017 Fall</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Course:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="codeBox">
       <item>
        <property name="text">
         <string>BİL101</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BİL105</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BİL110</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ENG199</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FİZ103</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FİZ105</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MAT151</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ORY100</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TÜRK101</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BİL122</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BİL124</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BİL172</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FİZ104</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FİZ110</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MAT152</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MAT210</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TÜRK102</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BİL300</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BİL324</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BİL331</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bil240</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BİL343</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BİL344</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BİL367</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ENG330</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SOS204</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="gradeLbl">
       <property name="text">
        <string>Grade: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <item>
        <property name="text">
         <string>A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>A-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>C+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>C</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>C-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>D+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>D</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>F</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="creditLbl">
       <property name="text">
        <string>Credit:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="creditBox">
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
